doc/source/config.rst

```diff
@@ -37,6 +37,45 @@ several config files and pick from them using `-c`. If you only wish to control
 the specific tests that are to be run (and not their parameters) then using
 `-s` or `-t` on the command line may be more appropriate.
 
+Also you can configure bandit via
+`pyproject.toml <https://www.python.org/dev/peps/pep-0518/>`_ file. In this
+case you would explicitly specify the path to configuration via `-s` too.
+For example:
+
+.. code-block:: python
```

Comment:
We can specify the code block to be of type toml. This will provide better syntax highlighting.

```suggestion
.. code-block:: TOML
```

Reply:
Good catch. At the moment when I made PR, there was no TOML support in pygments 🙃